CHANGELOG.md

```diff
@@ -6,13 +6,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 This project does not follow SemVer, since modules are independent of each other; thus, SemVer does not make sense. Changes are grouped per module.
 
-## [unreleased]
+## [v2022.08.03]
+
 ## Global Accelerator
 - added a new module to launch an AWS Global Accelerator, [#104](https://github.com/dbl-works/terraform/pull/104)
 
 ## ElastiCache
-- added `data_tiering_enabled` to the available options, defaulting to `false`
-- added `name` as optional parameter in case multiple Redis-Clusters need to be launched for the same project/environment
+- added `data_tiering_enabled` to the available options, defaulting to `false`, [#100](https://github.com/dbl-works/terraform/pull/100)
+- added `name` as optional parameter in case multiple Redis-Clusters need to be launched for the same project/environment, [#100](https://github.com/dbl-works/terraform/pull/100)
+
+## ECS iam management
+- added multiple new modules under `./iam` for sane access controll, [#95](https://github.com/dbl-works/terraform/pull/95)
```

Comment:
typo: `controll` => `control`

Can we put this under iam management since it contains iam management that is not related to ECS as well. (ECS fix can be under ## ECS)